CHANGELOG.rst

```diff
@@ -4,6 +4,12 @@ Changelog
 `Unreleased <https://github.com/staticjinja/staticjinja/compare/main>`_
 -----------------------------------------------------------------------
 
+Changed
+^^^^^^^
+
+* Drop support for Python 3.6. See
```

Comment:
Can you actually phrase this as "Stop testing for Python 3.6" It still might work, it's just not officially supported.